**Задача 3**

**Известно, что рост футболистов в сборной распределен нормально с дисперсией генеральной совокупности, $\sigma^2=25\text{ кв.см.}$ Объем выборки $n=27$, среднее выборочное $M=174.2$. Найдите доверительный интервал для математического ожидания с $\alpha=0,95$**

Поскольку нам известнa дисперсия генеральной совокупности $\sigma^2=25$, то для расчета $95\%$-го доверительного интервала, будем использовать $Z$-критерий. И границы доверительного интервала расчитаем по формуле:  
$\displaystyle\bar X\pm z_{\alpha/2}\cdot\frac{\sqrt{\sigma^2}}{\sqrt n}$ , где:  
$\bar X=M=174.2$-среднее значение выборки    
$z_{\alpha/2}$ табличное значение $Z$-критерия, и для $95\%$-го доверительного интервала $z_{\alpha/2}=1,96$  
$\sigma^2=25$-дисперсия генеральной совокупности  
$n=27$-объем выборки  

In [1]:
left=174.2-(1.96*25**0.5)/27**0.5
right=174.2+(1.96*25**0.5)/27**0.5
print(f'95%-й доверительный интервал для оценки мат. ожидания генеральной совокупности: [{left: .4f};{right: .4f}].')

95%-й доверительный интервал для оценки мат. ожидания генеральной совокупности: [ 172.3140; 176.0860].


Ответ.  
95%-й доверительный интервал для оценки мат. ожидания генеральной совокупности - [ 172.3140; 176.0860]